src/devices/acurite.c

```diff
@@ -588,7 +588,7 @@ static int acurite_atlas_decode(r_device *decoder, bitbuffer_t *bitbuffer, unsig
         /* clang-format off */
         data = data_append(data,
                 "wind_dir_deg",     NULL,           DATA_FORMAT,    "%.1f",         DATA_DOUBLE, wind_dir,
-                "rain_in",          "Rainfall Accumulation", DATA_FORMAT, "%.2f in", DATA_DOUBLE, raincounter * 0.01f,
+                "rain_in",          "Rainfall Accumulation", DATA_FORMAT, "%.2f in", DATA_DOUBLE, raincounter * 0.01,
```

Comment:
In this case I actually wanted it to be a float but it seems to be missed from the C-standard. Or something like that.

Reply:
I'm going to interpret this as 1) you would have liked to have DATA_FLOAT as a format and used it  2) given that it says DOUBLE, this change is ok.   Of course speak up if that second point is a  misinterpretation; the first point doesn't really bear on "should we merge this".